res/quad.fs.glsl

```diff
@@ -8,11 +8,16 @@ bool PointInRect(vec2 p, vec2 p0, vec2 p1)
 
 void main(void)
 {
-    // Clip out rect
+    // Clip out.
     if (PointInRect(vPosition, vClipOutRect.xy, vClipOutRect.zw)) {
         discard;
     }
 
+    // Clip in.
+    if (vClipInRect != vec4(0.0) && !PointInRect(vPosition, vClipInRect.xy, vClipInRect.zw)) {
```

Comment:
I don't think you need this if () condition - I'm fairly sure it wasn't in the shader last time?